pkg/apis/settings/settings.go

```diff
@@ -65,37 +64,15 @@ func (*Settings) Inject(ctx context.Context, cm *v1.ConfigMap) (context.Context,
 }
 
 func (in *Settings) Validate() (err error) {
-	if in.BatchMaxDuration == nil {
-		err = multierr.Append(err, fmt.Errorf("batchMaxDuration is required"))
-	} else if in.BatchMaxDuration.Duration <= 0 {
-		err = multierr.Append(err, fmt.Errorf("batchMaxDuration cannot be negative"))
+	if in.BatchMaxDuration < 1 {
```

Comment:
this should be `time.Second`. IIRC, this actually means 1 microsecond, which isn't what we want here.